blazor/mention/getting-started.md

```diff
@@ -0,0 +1,317 @@
+---
+layout: post
+title: Getting Started with Blazor Mention Component | Syncfusion
```

Comment:
Getting started added by core team for Mention - https://github.com/syncfusion-content/blazor-docs/pull/1788